Makefile

```diff
@@ -164,6 +164,14 @@ evg-test-load-balancers:
 evg-test-kms:
 	go test -exec "env PKG_CONFIG_PATH=$(PKG_CONFIG_PATH) LD_LIBRARY_PATH=$(LD_LIBRARY_PATH)" $(BUILD_TAGS) -v -timeout $(TEST_TIMEOUT)s ./mongo/integration -run TestClientSideEncryptionProse/kms_tls_tests >> test.suite
 
+.PHONY: evg-test-kmip
+evg-test-kmip:
```

Comment:
No need to change now. These tests are not specific to KMIP. I think `evg-test-kmip` with `evg-test-kms` can be combined in a future PR.